agent/host.go

```diff
@@ -28,6 +28,8 @@ import (
 	"os"
 	"strings"
 
+        "github.com/Jaganathancse/numautils"
```

Comment:
You need to add a licence to your project

Reply:
will update it.